aten/src/ATen/native/quantized/cpu/UpSampleBilinear2d.cpp

```diff
@@ -152,19 +152,33 @@ Tensor upsample_bilinear2d_quantized_cpu(
       "It is expected output_size equals to 2, but got size ",
       output_size.size());
 
+  int64_t channels = input.size(1);
```

Comment:
You need to check the number of dimensions before accessing the dimension 1 here